src/Evolution/Initialization/Interface.hpp

```diff
@@ -48,39 +72,54 @@ namespace Actions {
 ///         Directions, Tags::Normalized<Tags::UnnormalizedFaceNormal<Dim>>>`
 /// - Removes: nothing
 /// - Modifies: nothing
-template <typename System>
+template <typename System, typename SliceTagsToFace,
+          typename SliceTagsToExterior,
+          typename FaceComputeTags = face_compute_tags<>,
+          typename ExteriorComputeTags = exterior_compute_tags<>>
```

Comment:
I don't agree with any of what you said. I think the issue is more subtle because like you said, it depends on the scheme, etc. and **also** on the order that things need to be added. Again, I'm making the minimal change to allow Burgers, scalar wave, GRMHD, and (hopefully) Newtonian Euler to use the same code.

Reply:
I just fear this will be abused and users add long lists of sliced variables to the Metavariables instead of writing another initializer..

Reply:
In my opinion that isn't anymore abusive than having a `AddComputeItems` action. What you're calling abuse is what I sort of view as the intended behavior.

Reply:
we decided in a call to keep the current design for now and later split off initializers for numerical fluxes